docs/sources/release-notes/v2.4.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Grafana Mimir version 2.4 release notes"
+menuTitle: "V2.4 release notes"
+description: "Release notes for Grafana Mimir version 2.4"
+weight: 300
+---
+
+# Grafana Mimir version 2.4 release notes
+
+Grafana Labs is excited to announce version 2.4 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+**Note**: If you are upgrading from Grafana Mimir 2.3, review the [list of important changes](#important-changes) that follow.
+
+## Features and enhancements
+
+- **Query-scheduler ring-based service discovery**:
+  The [query-scheduler]({{< relref "../operators-guide/architecture/components/query-scheduler/index.md" >}}) is an optional, stateless component that retains a queue of queries to execute, and distributes the workload to available queriers. The use the query-scheduler, query-frontends and queriers are required to discover the addresses of the query-scheduler instances.
+
+  Other than the DNS-based service discovery, Mimir 2.4 introduces the ring-based service discovery for the query-scheduler. When enabled, the query-schedulers join their own hash ring (similar to other Mimir components), and the query-frontends and queriers discover query-scheduler instances via the ring.
```

Comment:
```suggestion
  In addition to DNS-based service discovery, Mimir 2.4 introduces the ring-based service discovery for the query-scheduler. When enabled, the query-schedulers join their own hash ring (similar to other Mimir components), and the query-frontends and queriers discover query-scheduler instances via the ring.
```
Do you mean to say, "As an alternative to"? I kind of think so, but you decide. :)